packages/react-core/src/demos/DateTimePicker.md

```diff
@@ -0,0 +1,106 @@
+import styles from '@patternfly/react-styles/css/components/DatePicker/date-picker';
+
+## Demos
+
+In this section, we demonstrate how to use a [CalendarMonth](/components/calendar-month) and [Dropdown](/components/dropdown) to compose a simple DateTimePicker.
+The input is read only meaning that the following example does not cover pasting a copied date and time. Also the time can be picked only from the predefined options and cannot be changed in the input.
+
+### Date and time picker
+
+```js
+import React from 'react';
+import { Dropdown, DropdownToggle, DropdownItem, CalendarMonth, InputGroup, TextInput, Button, Popover } from '@patternfly/react-core';
+import styles from '@patternfly/react-styles/css/components/DatePicker/date-picker';
+import OutlinedCalendarAltIcon from '@patternfly/react-icons/dist/esm/icons/outlined-calendar-alt-icon';
+import OutlinedClockIcon from '@patternfly/react-icons/dist/esm/icons/outlined-clock-icon';
+
+DateTimePicker = () => {
+  const [isCalendarOpen, setIsCalendarOpen] = React.useState(false);
+  const [isTimeOpen, setIsTimeOpen] = React.useState(false);
+  const [valueDate, setValueDate] = React.useState("YYYY-MM-DD");
+  const [valueTime, setValueTime] = React.useState("HH:MM");
+  const times = Array.from(new Array(10), (_, i) => i + 8);
+  const defaultTime = "0:00";
+  const dateFormat = date => date.toLocaleDateString('en-US', { year: 'numeric', month: '2-digit', day: '2-digit' }).replace(/\//g,'-');
+
+  const onToggleCalendar = () => {
+    setIsCalendarOpen(!isCalendarOpen);
+    setIsTimeOpen(false);
+  };
+
+  const onToggleTime = ev => {
+      setIsTimeOpen(!isTimeOpen);
+      setIsCalendarOpen(false);
+  };
+
+  const onSelectCalendar = newValueDate => {
+      const newValue = dateFormat(newValueDate);
+      setValueDate(newValue);
+      setIsCalendarOpen(!isCalendarOpen);
+      // setting default time when it is not picked
+      if (valueTime === "HH:MM") {
+          setValueTime(defaultTime);
+      }
+  };
+
+  const onSelectTime = ev => {
+      setValueTime(ev.target.value);
+      setIsTimeOpen(!isTimeOpen);
+  };
+
+  const timeOptions = times.map(time => (
+      <DropdownItem key={time} component="button" value={`${time}:00`}>
+        {`${time}:00`}
+      </DropdownItem>
+  ))
+  const calendar = (
+      <CalendarMonth
+        date={valueDate}
+        onChange={onSelectCalendar}
+      />
+  );
+  const time = (
+      <Dropdown
+        onSelect={onSelectTime}
+        toggle={
+          <DropdownToggle toggleIndicator={null} onToggle={onToggleTime}>
+            <OutlinedClockIcon />
+          </DropdownToggle>
+        }
+        isOpen={isTimeOpen}
+        dropdownItems={timeOptions}
+    />
+  );
+  const calendarButton = (
+      <Button variant="control" onClick={onToggleCalendar}>
+          <OutlinedCalendarAltIcon />
+      </Button>
+  );
+  return (
+    <div className={styles.datePicker}>
```

Comment:
Let's remove this use of `styles` and avoid that import. 
If you want to limit the width of the input, let's just use a `style={{width: '250px' }}` or something like that instead.
Generally, consumers are not aware of our style class name imports.